modules/cart/commerce_cart.module

```diff
@@ -223,3 +247,19 @@ function commerce_cart_views_data_alter(array &$data) {
 function template_preprocess_commerce_cart_block(&$variables) {
   $variables['attributes']['class'][] = 'cart--cart-block';
 }
+
+/**
+ * Implements hook_cron().
+ */
+function commerce_cart_cron() {
+  $queue = \Drupal::queue('cart_expiries');
+
+  $ids = \Drupal::entityQuery('commerce_order')
+    ->notExists('completed')
```

Comment:
Since we're putting this in cart, we need to check for things flagged a cart (not just drafts.) So let's change this to `->condition('cart', TRUE)`